integrations/os2sync/lcdb_os2mo.py

```diff
@@ -0,0 +1,200 @@
+#
+# Copyright (c) 2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+from sqlalchemy.orm import sessionmaker
+
+from exporters.sql_export.lc_for_jobs_db import get_engine  # noqa
+from exporters.sql_export.sql_table_defs import (KLE, Adresse, Bruger,
+                                                 Engagement, Enhed,
+                                                 ItForbindelse)
+from integrations.os2sync import config, os2mo
+
+settings = config.settings
+logger = logging.getLogger(config.loggername)
+
+
+def get_session(engine):
+    return sessionmaker(bind=engine, autoflush=False)()
+
+
+def has_kle():
+    # nothing in db if no kle
+    return True
+
+
+scope_to_scope = {
+    "E-mail": "EMAIL",
+    "Telefon": "PHONE",
+    "DAR": "DAR",
+    "EAN": "EAN",
+    "P-nummer": "PNUMBER",
+    "Text": "TEXT",
+    "Ansvarlig": "ANSVARLIG",
+    "Udførende": "UDFOERENDE",
+    "Indsigt": "INDSIGT",
+}
+
+
+def get_sts_user(session, uuid, allowed_unitids):
+    base = session.query(Bruger).filter(Bruger.uuid == uuid).one()
+
+    sts_user = {
+        "Uuid": uuid,
+        "UserId": uuid,
+        "Positions": [],
+        "Person": {"Name": base.fornavn + " " + base.efternavn, "Cpr": base.cpr},
+    }
+    if not settings["OS2SYNC_XFER_CPR"]:
+        sts_user["Person"]["Cpr"] = None
+
+    addresses = []
+    for lc_address in session.query(
+        Adresse
+    ).filter(Adresse.bruger_uuid == uuid).all():
+        address = {
+            "address_type": {
+                "uuid": lc_address.adressetype_uuid,
+                "scope": scope_to_scope[lc_address.adressetype_scope]},
+            "name": lc_address.værdi,
+            "value": lc_address.dar_uuid,
+            "uuid": lc_address.uuid  # not used currently
+        }
+        addresses.append(address)
+    os2mo.addresses_to_user(sts_user, addresses)
+
+    engagements = []
+    for lc_engagement in session.query(
+        Engagement
+    ).filter(Engagement.bruger_uuid == uuid).all():
+        engagements.append({
+            "uuid": lc_engagement.uuid,
+            "org_unit": {"uuid": lc_engagement.enhed_uuid},
+            "job_function": {"name": lc_engagement.stillingsbetegnelse_titel}
+        })
```

Comment:
```suggestion
    def construct_engagement(lc_engagement):
        return {
            "uuid": lc_engagement.uuid,
            "org_unit": {"uuid": lc_engagement.enhed_uuid},
            "job_function": {"name": lc_engagement.stillingsbetegnelse_titel}
        }
    sql_engagements = session.query(Engagement).filter(Engagement.bruger_uuid == uuid).all()
    engagements = list(map(construct_engagement, sql_addresses))
```
Jeg ville nok have skrevet dette som et map, så man ikke behøver tjekke om der sker noget andet spændende i den her for løkke eller om det bare er et map.

Reply:
Du har for meget tid, Emil :-)